services/src/main/java/org/keycloak/protocol/ciba/decoupledauthn/DelegateDecoupledAuthenticationProvider.java

```diff
@@ -170,7 +167,11 @@ public void doBackchannelAuthentication(ClientModel client, BackchannelAuthentic
                 userSessionIdWillBeCreated, userIdToBeAuthenticated, client.getClientId(), authResultId);
         String decoupledAuthId = persistDecoupledAuthId(session, decoupledAuthIdData, expiresIn);
 
+        String userCodeSupported = client.getAttribute(OIDCConfigAttributes.BACKCHANNEL_USER_CODE_PARAMETER_REQUIRED);
```

Comment:
It might be better to use `OIDCAdvancedConfigWrapper` rather that use `ClientModel.getAttribute` directly.

Reply:
Changed this, thank you.

Reply:
I've confirmed. Thank you.